agent/win-agent/build-image.ps1

```diff
@@ -0,0 +1,21 @@
+$IMAGE=$(get-item env:IMAGE -ErrorAction Ignore)
+
+Invoke-WebRequest -Uri "http://dev-tool.oss-cn-shenzhen.aliyuncs.com/windows/devcon.exe" -OutFile devcon.exe
```

Comment:
Is it possible to get this hosted on github/rancher versus aliyun?

Reply:
Is it ok that I just put it under `./agent-windows`? And we don't event need to download it or validate it.